cmd/libsnap-confine-private/mount-opt-test.c

```diff
@@ -211,10 +211,13 @@ static bool broken_mount(struct sc_fault_state *state, void *ptr)
 	return true;
 }
 
-static void test_sc_do_mount(void)
```

Comment:
Can you please spin this off into a separate PR and discuss with @jdstrand when he's back in two weeks?